app/assets/javascripts/cable.js

```diff
@@ -0,0 +1,18 @@
+//= require action_cable
+
+$(function () {
+  var cable = ActionCable.createConsumer('/ws/notifications');
```

Comment:
This will try creating the connection before `ws_init` has run.. Is that correct?